runelite-mixins/src/main/java/net/runelite/mixins/RSActorMixin.java

```diff
@@ -57,6 +57,9 @@
 	@Shadow("clientInstance")
 	private static RSClient client;
 
+	@Inject
+	private int index;
```

Comment:
This index is ambiguous on Actor since it indexes two different arrays depending on what type of Actor it is... I would just put it in Player.